app/controllers/admin/articles_controller.rb

```diff
@@ -45,9 +45,8 @@ def update
       article.boosted_additional_articles = article_params[:boosted_additional_articles].to_s == "true"
       article.boosted_dev_digest_email = article_params[:boosted_dev_digest_email].to_s == "true"
       article.user_id = article_params[:user_id].to_i
-      article.second_user_id = article_params[:second_user_id].to_i
-      article.third_user_id = article_params[:third_user_id].to_i
-      article.update!(article_params)
+      article.co_author_ids = article_params[:co_author_ids].split(/\s*,\s*/)
```

Comment:
I think we should change this

`"1,2, 3 ".split(/\s*,\s*/)` results in `["1", "2", "3 "]` which brings along both a 3 with a space

What about:

```ruby
ids.split(",").map(&:strip)
```

This way we remove the leading and trailing spaces for each value and let the validation occur